app/main.py

```diff
@@ -1 +1,28 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cmd = old_file = new_file = None
+    try:
+        cmd = command.split()[0]
+        old_file = command.split()[1]
+        new_file = command.split()[2]
+    except IndexError:
+        print(f"You don't put all necessary data"
+              f"{{'cmd': {cmd},"
+              f"'old_file': {old_file},"
+              f"'new_file': {new_file}}}")
```

Comment:
You can simplify this code by using `try` with `cmd, old_file, new_file = command.split()` and `ValueError`